api/src/main/java/org/openmrs/api/OrderService.java

```diff
@@ -853,4 +854,97 @@ public Order discontinueOrder(Order orderToDiscontinue, String reasonNonCoded, D
 	 */
 	@Authorized(PrivilegeConstants.GET_ORDERS)
 	public List<OrderGroup> getOrderGroupsByEncounter(Encounter encounter) throws APIException;
+	
+	/**
+	 * Return all OrderGroupAttributeTypes
+	 *
+	 * @return all {@link OrderGroupAttributeType}s
+	 * @should return all orderGroup attribute types including retired ones
+	 */
+	@Authorized(PrivilegeConstants.GET_ORDERS)
+	 List<OrderGroupAttributeType> getOrderGroupAttributeTypes() throws APIException;
+	
+	/**
+	 * Fetches OrderGroupAttributeType using provided Id
+	 * 
+	 * @param id The Id of the OrderGroupAttributeType to fetch from the database
+	 * @return the {@link OrderGroupAttributeType} with the given internal id
```

Comment:
Same as above in all these.

Reply:
Removed the extra line between the class and  "String PARALLEL_ORDERS "